jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -826,54 +812,141 @@ def _make_div_rem_harness(name, *, shapes=((2,), (2,)), dtype=np.float32,
   ]
 )
 
-_LAX_BINARY_ELEMENTWISE = (
-  lax.add, lax.atan2, lax.div, lax.igamma, lax.igammac, lax.max, lax.min,
-  lax.nextafter, lax.rem, lax.sub)
+def _make_binary_elementwise_harness(name, *, prim, shapes=((20, 20), (20, 20)),
+                                     dtype):
+  lhs_shape, rhs_shape = shapes
+  return Harness(f"{name}_lhs={jtu.format_shape_dtype_string(lhs_shape, dtype)}_rhs={jtu.format_shape_dtype_string(rhs_shape, dtype)}",
+                 prim.bind,
+                 [RandArg(lhs_shape, dtype), RandArg(rhs_shape, dtype)],
+                 prim=prim,
+                 dtype=dtype,
+                 shapes=shapes)
 
-lax_binary_elementwise = tuple(
-  Harness(f"{f_lax.__name__}_{jtu.dtype_str(dtype)}",
-          f_lax,
-          [arg1, arg2],
-          lax_name=f_lax.__name__,
-          dtype=dtype
-          )
-  for f_lax in _LAX_BINARY_ELEMENTWISE
+lax_atan2 = tuple( # Validate dtypes
+  _make_binary_elementwise_harness("dtypes", prim=lax.atan2_p, dtype=dtype)
   for dtype in jtu.dtypes.all_floating
-  for arg1, arg2 in [
-    (np.array([-1.6, -1.4, -1.0, 0.0, 0.1, 0.2, 1., 1.4, 1.6], dtype=dtype),
-     np.array([-1.6, 1.4, 1.0, 0.0, 0.1, 0.2, 1., 1.4, -1.6], dtype=dtype))
+) + tuple( # Validate broadcasting
+  _make_binary_elementwise_harness("broadcasting", prim=lax.atan2_p,
```

Comment:
Is there a way to consolidate the broadcasting tests so we don't have to repeat this pattern?

Reply:
Hm I believe yes. This can probably be factorised a bit better – I maybe overly used `sed` :)
I think maybe `_make_binary_elementwise_harness` could become a function that only takes a primitive and a set of dtypes, and returns the whole tuple for both dtypes/broadcasting checks.
WDYT?

Reply:
Sounds reasonable.